actors/builtin/miner/monies.go

```diff
@@ -64,15 +73,18 @@ func PledgePenaltyForTermination(initialPledge abi.TokenAmount, sectorAge abi.Ch
 // newly-committed power, holding the per-epoch block reward constant (though in reality it will change over time).
 // The network total pledge and circulating supply parameters are currently unused, but may be included in a
 // future calculation.
-func InitialPledgeForPower(qaPower abi.StoragePower, networkQAPower abi.StoragePower, networkTotalPledge abi.TokenAmount, epochTargetReward abi.TokenAmount, networkCirculatingSupply abi.TokenAmount) abi.TokenAmount {
+func InitialPledgeForPower(qaPower abi.StoragePower, networkQAPower, baselinePower abi.StoragePower, networkTotalPledge abi.TokenAmount, epochTargetReward abi.TokenAmount, networkCirculatingSupply abi.TokenAmount) abi.TokenAmount {
 	// Details here are still subject to change.
 	// PARAM_FINISH
 	// https://github.com/filecoin-project/specs-actors/issues/468
-	_ = networkCirculatingSupply // TODO: ce use this
-	_ = networkTotalPledge       // TODO: ce use this
+	_ = networkTotalPledge // TODO: ce use this
 
-	if networkQAPower.IsZero() {
-		return epochTargetReward
-	}
-	return big.Mul(InitialPledgeFactor, ExpectedDayRewardForPower(epochTargetReward, networkQAPower, qaPower))
+	ipBase := big.Mul(InitialPledgeFactor, ExpectedDayRewardForPower(epochTargetReward, networkQAPower, qaPower))
+	additionalIPNum := big.Mul(big.Mul(LockTargetFactorNum, networkCirculatingSupply), qaPower)
```

Comment:
nit: In order to keep this consistent with the way the math is described, it would help to define `lockTargetNum` as in the inner multiplication here. Then `additionalIPNum` could be defined in terms of it. There's probably no need to redefine `LockTargetFactorDenom`.